src/pages/events/project-setup.md

```diff
@@ -1,58 +1,179 @@
 
-    ![select io management api](../_images/adobe-io-events.png)
+1. In the Stage workspace, click the **Add service** pop-up menu and select **API**.
 
-1. Select the **Generate a key pair** option and click **Generate keypair**. A `config.zip` file, which contains a `certificate_pub.crt` and `private.key` file, will automatically download.
+1. On the **Add an API** page, filter by Adobe Services, select **I/O Events**, and click **Next**.
 
-    ![generate a key pair](../_images/generate-key-pair.png)
+   ![Select IO events](../_images/adobe-io-events.png)
 
-    **Note**: If you want to manually [create a public key certificate], you can select the **Upload your public key** option.
+1. Select the **Generate a key pair** option and click **Generate keypair**. The `config.zip` file downloads automatically.
+
+   ![generate a key pair](../_images/generate-key-pair.png)
+
+   **Note**: If you want to manually [create a public key certificate](https://developer.adobe.com/developer-console/docs/guides/authentication/JWT/JWTCertificate/), you can select the **Upload your public key** option.
 
 1. Click **Save configured API**.
 
-## Download workspace configuration
+1. Unzip the downloaded `config.zip` file. The extracted `config` directory should contain a `certificate_pub.crt` and a `private.key` file.
+
+## Download the workspace configuration file
 
 To download a `.json` file containing your workspace configuration:
 
-1. Open the corresponding project on the [Adobe Developer Console].
+1. Open the corresponding project on the Adobe Developer Console.
 
 1. Select the **Project Overview** and click the **Download** button.
 
-    ![download the workspace config](../_images/download-workspace-config.png)
+   ![download the workspace config](../_images/download-workspace-config.png)
+
+   The `<Workspace-name>.json` file downloads automatically.
+
+## Set up App Builder and define a runtime action
+
+The first step to setting up your App Builder template is to set up your environment and crete a runtime action. For details about this process, see [Setting up Your Environment](https://developer.adobe.com/runtime/docs/guides/getting-started/setup/).
+
+1. Log in to Adobe IO:
+
+   ```bash
+   aio login
+   ```
+
+   Your web browser displays the log in page. Enter your Adobe ID credentials.
+
+1. Return to your terminal and navigate to where you want to initialize your project. Enter the following command:
+
+   ```bash
+   aio app init
+   ```
+
+   The terminal prompts you to select the path to your workspace.
+
+1. Select your project's organization.
+
+1. Select your project.
+
+1. Select the  **DX Experience Cloud SPA** option. The command then initializes a project with a default UI and creates a default Adobe I/O Runtime Action with a name similar to `dx-excshell-1/__secured_generic-<organization-id>-<project-name>-<workspace-name>`. The event configuration process requires that at least one action be defined.
```

Comment:
I'm definitely seeing the created action having this name when doing the step referenced on line 175. Just noting that when I run `aio app run` to launch App Builder, in the default UI the action seems to be referred to as just `dx-excshell-1/generic`

Reply:
Added the note below.